pkg/util/membercluster_client.go

```diff
@@ -111,8 +111,11 @@ func NewClusterDynamicClientSetForAgent(clusterName string, client client.Client
 	return &clusterClientSet, nil
 }
 
-func buildClusterConfig(clusterName string, client client.Client) (*rest.Config, error) {
-	cluster, err := GetCluster(client, clusterName)
+// BuildClusterConfig return rest config for member cluster.
```

Comment:
```suggestion
// BuildClusterConfig returns the rest config for the member cluster.
```

Reply:
I guess the benefits of using the `getter` thing could make it easier for testing. So, can we add a unit test here?

Reply:
It's covered by `TestNewClusterClientSet` and `TestNewClusterDynamicClientSet`. It seems redundantly~